### PR TITLE
zeroex - refactor poly

### PR DESCRIPTION
**Pull request by RantumBits**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

File: h origin -d bnb2 bnbB bnbuni brew chainflag chainflagPoly chainflageth cln contract dedup
Comment (Hosuke) [inline h origin -d bnb2 bnbB bnbuni brew chainflag chainflagPoly chainflageth cln contract dedup:1]: I think we may need to remove this file from the PR.

File: h origin -d fixc fixnolimits ftm ftm-mim layer_1_btc_view layer_1_eth_view main matcha-fee
Comment (Hosuke) [inline h origin -d fixc fixnolimits ftm ftm-mim layer_1_btc_view layer_1_eth_view main matcha-fee:1]: Same as above.

File: seeds/zeroex/polygon/_schema.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+version: 2
+
+seeds:
```
Comment (Hosuke) [inline seeds/zeroex/polygon/_schema.yml:3]: We are missing `zeroex_polygon_api_fills_deduped_sample` definition here.

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: seeds/zeroex/polygon/zeroex_polygon_api_fills_deduped_sample.csv [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+polygon,2023-01-01,2023-01-01 07:59,USDT,0x79375c41d88f839f551457145066096c5c8944bc,,0.4985,,498500,8.796288564730349e+18,0.498831004,0xc2132d05d31c914a87c6611c10748aeb04b58e8f,0x79375c41d88f839f551457145066096c5c8944bc,0x2f2dd99235cb728fc79af575f1325eaa270f0c99,0x00000002517569636b5377617000000000000000,0x1000000000000000000000000000000000000011,0x00b07f278afaa17ea587d7f8aedaa233337e02a309aff6f69c0d9535634c6bbb,0x1d8535a03710b0341ed52ddbdfd57fb0890d4377,0xd1ca1f4dbb645710f5d5a9917aa984a47524f49a,113,BridgeFill,true,1
+polygon,2023-01-01,2023-01-01 13:32,USDT,USDC,USDC-USDT,0.009999,0.009991,9999,9991,0.009996804771,0xc2132d05d31c914a87c6611c10748aeb04b58e8f,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,0xfd896db057f260adce7fd1fd48c6623e023406cd,0x0000000241706553776170000000000000000000,0x1000000000000000000000000000000000000011,0x00b6cdd803a3e1ca2126472a2dd5acb361f0d70a748997b4cbce07edc1affa25,0xfdc8db8dcd2aad69e0943ca6b0986a69a167a9e2,0xfd896db057f260adce7fd1fd48c6623e023406cd,447,BridgeFill,true,1
```
Comment (Hosuke) [inline seeds/zeroex/polygon/zeroex_polygon_api_fills_deduped_sample.csv:5]: Seems the `taker` and `maker` fields are not matched with the query results.
Do you want to recheck the seed here?

**Comment by jeff-dude** [comment]
thanks @Hosuke for helping pick this up 🙏

File: seeds/zeroex/polygon/zeroex_polygon_api_fills_deduped_sample.csv [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
-polygon,2023-01-01T00:00:00Z,2023-01-01T02:20:45Z,0xc168e40227e4ebd8c1cae80f7a55a4f0e6d66c97,USDC,,,41.3089,1.8752344479089898e+21,4.13089e+07,41.331206806,0xc168e40227e4ebd8c1cae80f7a55a4f0e6d66c97,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,0xc590175e458b83680867afd273527ff58f74c02b,0x000000024466796e000000000000000000000000,0x11ededebf63bef0ea2d2d071bdf88f71543ec6fb,0x00ca8b7d0039030fd1d08480fb5f5c7f7edb47a6e40fee0644b3e05ce01c8fb4,0x4b1fa071e31836f61bf26b10bf4d577345fddcf6,0x1a1ec25dc08e98e5e93f1104b5e5cdd298707d31,147,BridgeFill,true,1
\ No newline at end of file
+polygon,2023-01-01,2023-01-01 14:16,WMATIC,0x6948bad33a52ab0bed5cd0bc867fb091c3d920c7,,3.8250945851720997,,3.8250945851720996e+18,5.050003892548732e+19,2.8917705005779997,0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270,0x6948bad33a52ab0bed5cd0bc867fb091c3d920c7,0xf0074d056d005c0696e699ef5312111750835885,0x00000012556e6973776170563300000000000000,0x1000000000000000000000000000000000000011,0x00393279ae6ed27158a7b2281d953bf34b3027535a7b4a677e8451d1a47e9ead,0xf0074d056d005c0696e699ef5312111750835885,0xdef1c0ded9bec7f1a1670819833240f027b25eff,236,BridgeFill,true,2
```
Comment (Hosuke) [inline seeds/zeroex/polygon/zeroex_polygon_api_fills_deduped_sample.csv:2]: Seems this line is having mismatched `evt_index` and `maker`:
https://dune.com/queries/3040806

Comment (Hosuke) [reply seeds/zeroex/polygon/zeroex_polygon_api_fills_deduped_sample.csv:2]: I will modify the seed based on this query result:
https://dune.com/queries/3040835

Currently only 2 records with tx_hash in the query results.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @RantumBits 

I adjusted seeds in seeds/zeroex/polygon/zeroex_polygon_api_fills_deduped_sample.csv to match query results.

[Sample zeroex_polygon_api_fills_deduped](https://dune.com/queries/3041494)
[Sample zeroex_polygon_api_fills](https://dune.com/queries/3041571)

**Comment by Hosuke** [comment]
I am fixing the topic index shifting and going to retry the original seed records.

File: seeds/zeroex/polygon/zeroex_polygon_api_fills_deduped_sample.csv [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
+polygon,0x API,2023-01-01,2023-01-01 7:02,USDC,DAI,DAI-USDC,98.247675,98.24887947,98247675,98248879472596880000,98.28906326,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,0x8f3cf7ad23cd3cadbd9735aff958023239c6a063,0x0e5789f77945013c3acfc1cee3e1e03d2dd8b8c5,0x0000001942616c616e6365725632000000000000,0x11ededebf63bef0ea2d2d071bdf88f71543ec6fb,0x01615a9f260c70c4e2c2e2e65ea21df56b3bd17034f694cf0525721e1ceaa0d1,0x0e5789f77945013c3acfc1cee3e1e03d2dd8b8c5,0x1a1ec25dc08e98e5e93f1104b5e5cdd298707d31,227
+polygon,0x API,2023-01-01,2023-01-01 0:19,USDC,CRV,CRV-USDC,50,95.0201732,50000000,95020173202461750000,50.00704359,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,0x172370d5cd63279efa6d502dab29171933a610af,0xaf677145f8f03ecd25da06e290c59cb5a64aa163,0x00000012556e6973776170563300000000000000,0x5bd68b4d6f90bcc9f3a9456791c0db5a43df676d,0x02c241e84f97f5389550c94b46ee9187a846401cf887e0d42002bb4e1c458c10,0xaf677145f8f03ecd25da06e290c59cb5a64aa163,0xdef1c0ded9bec7f1a1670819833240f027b25eff,94
```
Comment (Hosuke) [inline seeds/zeroex/polygon/zeroex_polygon_api_fills_deduped_sample.csv:3]: Event 94 in this transaction is exchange of WETH and CRV:
https://polygonscan.com/tx/0x02c241e84f97f5389550c94b46ee9187a846401cf887e0d42002bb4e1c458c10#eventlog

I will update the taker token address.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @RantumBits 

[Sample zeroex polygon api fills](https://dune.com/queries/3046345)
[Sample zeroex polygon api fills deduped]()

**Review by jeff-dude — APPROVED** [review]
thanks again to you both 🙌

**Pull request merged**